AdrianBarros/assignments/treesBFS/lc103/lc103.py

```diff
@@ -0,0 +1,95 @@
+# Problem Statement #
+
+# Given a binary tree, populate an array to represent its zigzag level order traversal. You should populate the values of all nodes of the first level from left to right, then right to left for the next level and keep alternating in the same manner for the following levels.
+'''
+input: a bi tree
```

Comment:
Nice overview here. Don't forget to consider edge cases. Would also suggest writing additional test cases, especially for your edge cases.